man/systemd-mount.xml

```diff
@@ -64,7 +64,7 @@
     <cmdsynopsis>
       <command>systemd-umount</command>
       <arg choice="opt" rep="repeat"><replaceable>OPTIONS</replaceable></arg>
-      <arg choice="plain" rep="repeat"><replaceable>WHERE</replaceable></arg>
+      <arg choice="plain" rep="repeat"><replaceable>WHAT|WHERE...</replaceable></arg>
```

Comment:
much like the --help text this should probably be triplicated for the tree main ways to invoke it, i.e. for mounting, for listing and for unmounting